service/src/test/java/uk/gov/hmcts/reform/fpl/service/payment/PaymentServiceTest.java

```diff
@@ -229,13 +239,14 @@ void resetInvocations() {
         reset(paymentApi);
     }
 
-    private CreditAccountPaymentRequest.CreditAccountPaymentRequestBuilder testCreditAccountPaymentRequestBuilder() {
+    private CreditAccountPaymentRequest.CreditAccountPaymentRequestBuilder testCreditAccountPaymentRequestBuilder(
+        final String customerReference) {
         return CreditAccountPaymentRequest.builder()
             .accountNumber("PBA123")
             .caseReference("clientCode")
             .ccdCaseNumber(String.valueOf(CASE_ID))
             .currency(GBP)
-            .customerReference("fileReference")
+            .customerReference(paymentService.defaultCustomerReference(customerReference))
```

Comment:
Ah I see now, ok I don't like this approach, it causes thee linee too be ultimately not tested - we expect exactly what's returned by the service, we need to test it and provide the value manually.